### ISSUE TITLE
Cannot set isolation level in ibm_db_sa versions >= 0.3.5

### ISSUE THREAD

**Issue by singhshraddha** [ibmdb/python-ibmdbsa#96]
The issue is as fixed in this PR https://github.com/ibmdb/python-ibmdbsa/pull/79

stack trace from source to fail (engine is a sqlalchemy Engine)
```
    with engine.connect() as connection:
 /venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2272, in connect
    return self._connection_cls(self, **kwargs)
  File "/venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 126, in __init__
    self.dispatch.engine_connect(self, self.__branch)
  File "/venv/lib/python3.7/site-packages/sqlalchemy/event/attr.py", line 322, in __call__
    fn(*args, **kw)
  File "/venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 505, in set_isolation
    self._set_connection_isolation(connection, isolation_level)
  File "/venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 531, in _set_connection_isolation
    self.set_isolation_level(connection.connection, level)
  File "/venv/lib/python3.7/site-packages/ibm_db_sa/ibm_db.py", line 143, in set_isolation_level
    attrib = {SQL_ATTR_TXN_ISOLATION:_get_cli_isolation_levels(self,level)}
NameError: name '_get_cli_isolation_levels' is not defined
```

**Comment by amukherjee28** [comment]
Hi,

The PR is currently not merged currently that is why you might be seeing the issue.
I will have a look at this and update at the earliest.

Meanwhile can you provide a sample code that you are running to fetch the isolation level in iseries.

Are you using the I-series ODBC driver or are you using the direct connection to I-series.

Thanks.

**Comment by singhshraddha** [comment]
Here's the sample code I'm using to set the isolation level, it's passed through sqlalchmey's create_engine method and the isolation is fetch when a connection is made using the sqlalchemy engine (so, not a direct connection)

```
sqlalchemy_connection_string = 'db2+ibm_db://username:password@host:port/database;'
sqlalchemy_connection_kwargs = {'execution_options': {'isolation_level': "UR"}}
engine = create_engine(sqlalchemy_connection_string, **sqlalchemy_connection_kwargs))
```

**Comment by amukherjee28** [comment]
hi @singhshraddha 

Thank you for raising this to us.

I have tested the fix #79 

The same code has been merged to the git.
You can clone the same and install for you till we make another release.

Thanks.

**Comment by amukherjee28** [comment]
Please test it once after installing the code from the master branch and then let me know.

I have tested the same from my end.

Thanks.

**Comment by singhshraddha** [comment]
Hi @amukherjee28, I've tested the code off of the master branch and it works as expected. Let me know when this fix goes in a release

**Comment by amukherjee28** [comment]
New version is released and this issue had been addressed in the new release 0.3.7.

Please check and let me know if this can be closed.

**Comment by singhshraddha** [comment]
good to close on my end
